tests/playwright/assets/elements-utils.js

```diff
@@ -52,7 +52,17 @@
 	return `[data-id = "${ id }"]`;
 };
 
+const viewportSize = {
+	desktop: { width: 1920, height: 1080 },
+	tablet_extra_next: { width: 1201, height: 960 },
+	tablet_extra: { width: 1200, height: 960 },
+	tablet_next: { width: 1025, height: 960 },
+	tablet: { width: 1024, height: 960 },
+	mobile: { width: 400, height: 480 },
+};
```

Comment:
@hein-obox Maybe move this to the `enums` folder?

Reply:
Done